modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/linear-segregated/1d-scalar/tests

```diff
@@ -0,0 +1,23 @@
+[Tests]
+  issues = '#28819'
+  design = 'SIMPLE.md LinearFVScalarAdvection.md'
+  [channel]
+    requirement = "The system shall be able to solve a "
+    [steady-physics]
+      type = 'CSVDiff'
+      input = channel-conservation-physics.i
+      csvdiff = 'channel-conservation-physics_csv.csv'
+      recover = false # we don't support recovery for SIMPLE yet
+      max_threads = 1 # see libmesh issue #3808
+      detail = "1D steady scalar advection problem using the shorthand Physics syntax and show scalar conservation,"
+    []
+    [transient-physics]
+      type = 'CSVDiff'
+      input = channel-transient-physics.i
+      csvdiff = 'channel-transient-physics_csv.csv'
+      recover = false # we don't support recovery for SIMPLE yet
```

Comment:
I think for PIMPLE we support recovery, at least we should.

Reply:
SIMPLE is just a steady solver and right now only Steady is skipped for recovery.

Reply:
will change the wording now